### PR TITLE
feat: addition of descriptions and examples to dedicated

### PR DESCRIPTION
**Pull request by dimakis**
Closes # 

### Verification Steps

1. Do x
2. Do y

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation change
- [ ] Other (please specify)

### CODE REVIEW

File: pkg/core/localize/locales/en/cmd/dedicated.en.toml [hunk 1]
```diff
@@ -42,4 +42,19 @@ There will be N/3 streaming units in your Kafka cluster, where N is the machine
...
+
+[dedicated.longDescription]
+one = '''
+Red Hat OpenShift Streams for Apache Kafka allows you to use your own OpenShift clusters to provision your
+kafkas. These Kafka instances will be managed by Red Hat OpenShift Streams for Apache Kafka.
+'''
+
+[dedicated.example]
+one = '''
+# register-cluster, registers an OpenShift cluster with Red Hat OpenShift Streams for Apache Kafka.
```
Comment (rkpattnaik780) [inline pkg/core/localize/locales/en/cmd/dedicated.en.toml:58]:
```suggestion
# Register an OpenShift cluster with Red Hat OpenShift Streams for Apache Kafka
```

File: pkg/core/localize/locales/en/cmd/dedicated.en.toml [hunk 2]
```diff
@@ -42,4 +42,19 @@ There will be N/3 streaming units in your Kafka cluster, where N is the machine
...
+rhoas cluster register-cluster
```
Comment (rkpattnaik780) [inline pkg/core/localize/locales/en/cmd/dedicated.en.toml:59]:
```suggestion
rhoas dedicated register-cluster
```

**Review by rkpattnaik780 — CHANGES_REQUESTED** [review]
Dropping few suggestions. It is good to run `make generate-docs` when changes are made for command descriptions.

**Comment by dimakis** [comment]
@rkpattnaik780 all comments have been addressed... 😄

File: pkg/core/localize/locales/en/cmd/dedicated.en.toml [hunk 3]
```diff
@@ -42,4 +42,19 @@ There will be N/3 streaming units in your Kafka cluster, where N is the machine
 '''
 
 [dedicated.registerCluster.info.foundValidMachinePool]
-one = 'Using the valid machine pool:'
\ No newline at end of file
+one = 'Using the valid machine pool:'
+
+[dedicated.shortDescription]
+one = 'Manage your OpenShift clusters to use with Red Hat OpenShift Streams for Apache Kafka.'
```
[the remark this answers is not in the document]
Comment (rkpattnaik780) [reply pkg/core/localize/locales/en/cmd/dedicated.en.toml]: Remove the `.`

**Pull request closed without merging**